code/datums/ammo/rocket.dm

```diff
@@ -277,8 +277,9 @@
 
 /datum/ammo/rocket/custom
 	name = "custom rocket"
-	accurate_range = 8
-	max_range = 8
+	accuracy = HIT_ACCURACY_TIER_5
+	accurate_range = 7
+	max_range = 7
```

Comment:
wont this bring problems when shooting diagonally ??

Reply:
no